### PR TITLE
Ft navbar

### PR DESCRIPTION
**Pull request by nsengaqueen**
feat: implementation of the navbar section and it's responsiveness

### CODE REVIEW

File: src/sections/Header.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <p class="ml-14">About</p>
+    <p class="ml-8">Services</p>
+    <p>References</p>
+    <p class="ml-9">Contact</p>
+  </div>
+<span class="md:pt-4" >
+      <img
+    src="/Images/Group 4.png" 
+    class="pl-[4%] lg:hidden ml-12 md:ml-12 md:object-cover "
+  />
```
Comment (IGIRANEZAJosue) [inline src/sections/Header.html:23]: The navbar is not responsive on the desktop breakpoint.
Try using flex alignments instead of paddings and margins.

**Comment by nsengaqueen** [comment]
[Navbar - Responsiveness](https://trello.com/c/tVe2snCg/10-navbar-responsiveness)

**Comment by nsengaqueen** [comment]
[Navbar - Desktop](https://trello.com/c/Vu03csSg/9-navbar-desktop)

File: Images/Background Phone screen.svg
Comment (Algor-Fernand) [inline Images/Background Phone screen.svg:1]: Look into the naming of the files.

**Review by Algor-Fernand — CHANGES_REQUESTED** [review]
I have left some that you need to work on. But good job.

**Pull request closed without merging**